test/workers/test_websocket_worker.py

```diff
@@ -270,7 +270,7 @@ def test_evaluate(hook, start_proc):  # pragma: no cover
 
     dataset = sy.BaseDataset(data=data, targets=target)
 
-    kwargs = {"id": "evaluate_remote", "host": "localhost", "port": 8780, "hook": hook}
+    kwargs = {"id": "evaluate_remote", "host": "localhost", "port": 8790, "hook": hook}
```

Comment:
why the port change?

Reply:
are you kidding? 8790 is clearly vastly superior.

Reply:
I changed it in order to have a different port number than the ones used in the MNIST parallel demo. And yes, it's superior :-)